libass/ass_bitmap.c

```diff
@@ -98,12 +75,11 @@ void ass_synth_blur(const BitmapEngine *engine, Bitmap *bm,
 bool alloc_bitmap(const BitmapEngine *engine, Bitmap *bm,
                   int32_t w, int32_t h, bool zero)
 {
-    unsigned align = 1 << engine->align_order;
-    size_t s = ass_align(align, w);
+    size_t s = ass_align(ASS_ALIGNMENT, w);
```

Comment:
Use of the global alignment for bitmaps independent of the current asm mode? Indeed that looks like the simplest solution (I prefer `ASS_BITMAP_ALIGNMENT` though). Are your sure we don't want to check different alignments with checkasm later?

Reply:
After some thinking, I've concluded that the old approach with `BitmapEngine`-local alignments is better. Larger than necessary padding can have significant negative impact on performance in the case of many small bitmaps, which quite corresponds to subs rendering with its ton of font glyphs.